packages/fxa-auth-server/fxa-oauth-server/lib/keys.js

```diff
@@ -0,0 +1,75 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const config = require('./config');
+const JwTool = require('fxa-jwtool');
+
+const ISSUER = config.get('openid.issuer');
+
+function pub(key) {
+  // Hey, this is important. Listen up.
+  //
+  // This function pulls out only the **PUBLIC** pieces of this key.
+  // For RSA, that's the `e` and `n` values.
+  //
+  // BE CAREFUL IF YOU REFACTOR THIS. Thanks.
+  return {
+    kty: key.kty,
+    alg: 'RS256',
+    kid: key.kid,
+    'fxa-createdAt': key['fxa-createdAt'],
+    use: 'sig',
+    n: key.n,
+    e: key.e,
+  };
+}
+
+const PRIVATE_KEYS_RAW = [config.get('openid.key')];
+
+const oldPrivKey = config.get('openid.oldKey');
+if (oldPrivKey && Object.keys(oldPrivKey).length) {
+  PRIVATE_KEYS_RAW.push(oldPrivKey);
+}
+
+const SIGNING_KEY = JwTool.JWK.fromObject(config.get('openid.key'), {
+  iss: ISSUER,
+});
+
+const PUBLIC_KEYS_RAW = new Map();
+const VERIFICATION_PUBLIC_KEYS = new Map();
```

Comment:
Move `VERIFICATION_` as a suffix, or use a _JWK suffix instead for clarity.

Reply:
To clarify, these are "live" `PublicJwk` objects rather than just the JSON fields?

Reply:
Yes